Library/UIViewController+URLTests.swift

```diff
@@ -8,40 +8,6 @@ final class UIViewControllerURLTests: TestCase {
     XCTAssertEqual(UIViewController.supportedURLSchemes, ["http", "https"])
   }
 
-  func testGoToHttpScheme() {
```

Comment:
@dusi these tests now need to move to a UITest because we no longer run Library tests with a host application. I believe this is a more correct approach. Using `UIWindow` in a unit test is not allowed as it throws an exception:

```
NSInternalInconsistencyException: props must have a valid clientID
```